versions.tf

```diff
@@ -2,6 +2,9 @@ terraform {
   required_version = ">= 0.14, < 2.0"
 
   required_providers {
-    google-beta = "~> 3.75"
+    google = {
+      source  = "hashicorp/google"
+      version = ">= 3.75, < 5.0"
+    }
```

Comment:
i guess this is still only available in beta provider? so let's upgrade beta provider here...

Reply:
@mariux updated!